Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.3.2)
```

Comment:
Why the updates in this file? It seems to build without them.

I suggest that we remove the changes in this file.